src/style.css

```diff
@@ -1 +1,78 @@
-/* Styles go here */
+@font-face {
+  font-family: "Roboto";
+  src: url(fonts/Roboto-Medium.ttf);
+  font-style: normal;
+  font-weight: 500;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
```

Comment:
```suggestion
```
Box-sizing is the only property that we can set for all the element by `*` selector